### PR TITLE
Reconcile `az aro` with Azure/azure-cli

### PR DESCRIPTION
**Pull request by bennerv**
### Which issue this PR addresses:

Reconcile some changes that should've been in upstream az aro extension but were only in downstream (Azure/azure-cli)

### What this PR does / why we need it:
Same as above

### Test plan for issue:
N/A

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

File: python/az/aro/azext_aro/_help.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
     text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet
   - name: Create a cluster with 5 compute nodes and Red Hat pull secret.
     text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet --worker-count 5 --pull-secret @pullsecret.txt
+  - name: Create a Private cluster
```
Comment (jim-minter) [inline python/az/aro/azext_aro/_help.py:20]:
```suggestion
  - name: Create a private cluster
```

Comment (bennerv) [reply python/az/aro/azext_aro/_help.py]: Fixed

**Review by jim-minter — APPROVED** [review]
will merge on python tests green

**Pull request merged**